src/bootstrap.py

```
@@ -19,15 +19,12 @@
 
 
 # ---------------------------------------------------------------------------
-# LLM 層：從 providers.toml 建立 ResilientLLMService
+# LLM 層：從 DB provider config 建立 ResilientLLMService
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)